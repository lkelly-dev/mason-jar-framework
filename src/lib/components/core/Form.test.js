import React from 'react';
import { shallow } from 'enzyme';
import {
  Form,
  FormGroup,
  FormRow,
  FormInput,
  FormSelect,
  FormTextarea,
  FormLabel,
  FormUpload,
  FormDatepicker,
  FormToggle,
  FormCheck,
  FormSearchSelect,
} from "./Form";

describe('Form', () => {
  it('Form matches snapshot', () => {
    const component = shallow(<Form></Form>);
    expect(component).toMatchSnapshot();
  })
});
