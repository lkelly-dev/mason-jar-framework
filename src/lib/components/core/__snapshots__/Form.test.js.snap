// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Form matches snapshot 1`] = `<form />`;
